dotnet-desktop-guide/net/wpf/documents/snippets/how-to-print-xps-files/vb/MainWindow.xaml.vb

```diff
@@ -0,0 +1,129 @@
+﻿Imports System.IO
+Imports System.Printing
+Imports System.Threading
+
+Namespace CodeSampleVb
+    Partial Public Class MainWindow
+        Inherits Window
+
+        ''' <summary>
+        ''' Interaction logic for MainWindow.xaml
+        ''' </summary>
+        Public Sub New()
+
+            InitializeComponent()
+
+            ' Events.
+            AddHandler btnPrint.Click, AddressOf BtnPrint_Click
```

Comment:
The button click event should **not** be set here

Reply:
Done.